tests/closure/tclosure.nim

```diff
@@ -491,3 +492,10 @@ block tnoclosure:
       row = zip(row & @[0], @[0] & row).mapIt(it[0] + it[1])
     echo row
   pascal(10)
+
+block tcapturecdecl:
+  proc capture(p:proc(x:cint):cint {.cdecl.}) =
+    proc f() {.cdecl.} =
+      echo p(1)
```

Comment:
You capture `p` here, `p` is a parameter of `capture` and a stack location that will disappear. The calling convention of `p` is irrelevant, We could move `p` into an `environment` but then `f` cannot access the environment as it's `cdecl`.

Reply:
I thought that a `{.cdecl.}` calling convention uniquely identified a function pointer, my goal is to allow a function pointer to call another because afaik that is safe. Is there a better way to do that?

Reply:
I can only repeat myself at this point. I don't know why it would be "safe", it really isn't and I don't understand what a "uniquely identified function pointer" is and I don't understand why `cdecl` would be one but not e.g. `stdcall`.